code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -1092,6 +1092,10 @@ var/global/list/damage_icon_parts = list()
 		O.name = "[index]"
 		O.icon = t_inhand_state
 		O.icon_state = t_state
+		if(I.color)
+			O.color = I.color
+		else
+			O.color = "#ffffff"
```

Comment:
Just do `O.color = null`, it's clearer what you're trying to do, and should have the same effect, since null is white.